### ISSUE TITLE
typings for ToolbarMenuIcon and ToolbarIcon missing `icon` prop

### ISSUE THREAD

**Issue by rikkertkoppes** [jamesmfriedman/rmwc#333]
Using rmwc 3.0.1 with ts 3.0.3 and webpack 4.18.0

There is a ts compile issue with ToolbarMenuIcon and ToolbarIcon:

                    <ToolbarMenuIcon icon="menu" />
                    <ToolbarIcon icon="settings" />

These trigger a build error:

```
TS2339: Property 'icon' does not exist on type 'IntrinsicAttributes & { tag?: string | ComponentClass<any, any> | StatelessComponent<any>; wrap?: boolean; elementRef?: any; theme?: string | string[]; apiRef?: <S>(api: S) => S; } & HTMLAttributes<...> & HTMLProps<...> & { ...; }'.
```
Per the docs, I'd expect this to work. It seems that the icon prop is missing in the type declaration:

```
/**
 * A Menu Icon For the Toolbar. This is an instance of the Icon component.
 */
export declare const ToolbarMenuIcon: React.ComponentType<SimpleTagPropsT>;
/**
 * A standard Icon for toolbar actions. This is an instance of the Icon component.
 */
export declare const ToolbarIcon: React.ComponentType<SimpleTagPropsT>;
```
Where `SimpleTagPropsT` does not have an `icon` prop:

```
export declare type SimpleTagPropsT = {
    tag?: string | React.ComponentType<any>;
    wrap?: boolean;
    elementRef?: any;
    theme?: string | string[];
    apiRef?: <S>(api: S) => S;
} & React.HTMLAttributes<any> & React.HTMLProps<any>;
```

**Comment by rikkertkoppes** [comment]
I think it is a new issue, but I have not thoroughly checked. I briefly used v2 before upgrading to v3 and as I recall correctly, this was also a problem in that version.

I am using the `flexible` option of the toolbars and I have not yet seen something similar in `TopAppBar`

**Comment by jamesmfriedman** [comment]
Yeah, same here. Thats why I've kept it around, until it can have feature parity. Typing issue was a simple fix, pushing out 3.0.2 as a patch, look for it shortly.
